wagn/features/signup.feature

```diff
@@ -19,20 +19,21 @@ Feature: Signing up
     And I press "Submit"
     Then I should see "Signup Success"
     And "wanna@wagn.org" should receive an email with subject "verification link for My Wagn"
-    
+
     When I open the email
     And I click the first link in the email
     Then I should see "Wanna B"
-    
-    When I follow "Sign out"
+
+    When I go to the home page
+    And I follow "Sign out"
     Then I should not see "Wanna B"
```

Comment:
I'm wondering how this passed before (or why it started to fail with Rails 4 respectively). The link in the email leads to the "Wanna B" user card. When you click on "Sign out" there is still "Wanna B" in the header of the user card so the "I should not see 'Wanna B' " test fails.